modules/eco-poison-pill-operator-installation-cli.adoc

```diff
@@ -64,7 +61,7 @@ metadata:
     name: poison-pill-manager
     namespace: poison-pill
 spec:
-    channel: alpha
+    channel: stable
     name: poison-pill-manager
     source: redhat-operators
     sourceNamespace: openshift-marketplace
```

Comment:
![image](https://user-images.githubusercontent.com/58551443/155095285-7ea89836-edb3-45d8-8bbb-ebc9b579cf7a.png)
https://docs.openshift.com/container-platform/4.9/operators/user/olm-installing-operators-in-namespace.html#olm-installing-from-operatorhub-using-web-console_olm-installing-operators-in-namespace
Also, you can add to the subscription: installPlanApproval: Manual 
with comments from the screenshot about installPlanApproval